src/app/modules/catalog/components/catalog-promotion/catalog-promotion.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="catalog__promotion" *ngIf="isReadyToDisplay$ | async; else loading">
+    <ul
+        class="catalog__promotion-list"
+        *ngIf="promotions$ | async as promotions"
```

Comment:
it's confusing to see ngIf here as we have isReadyToDisplay$ above, so remove

Reply:
Why I can't use two ngIf? first ngIf for checking and displaying. The second one for getting data from Observable.